CHANGELOG.md

```diff
@@ -8,7 +8,15 @@ Added
 -
 
 * Add static partitioning in ArmoniK
-* Add the parameters `InitWorker__WorkerCheckRetries` and `InitWorker__WorkerCheckDelay` in the configmap of ArmoniK polling agent
+* Add the environment variable `Amqp__MaxRetries` in the configmap of ArmoniK core and Metrics exporter
+* Add the environment variables `ControlPlane__Partitions` and `ControlPlane__DefaultPartition` in the configmap of
+  ArmoniK control plane
+* Add the environment variable `ComputePlan__PartitionId` in the of ArmoniK polling agent container
```

Comment:
Not linked to keda and not really true

Reply:
They will be updated/deleted when we will change the partitioning integration